website/content/docs/secrets/databases/index.mdx

```diff
@@ -28,18 +28,28 @@ become invalid within a reasonable time of the lease expiring.
 
 ### Static roles
 
-The database secrets engine supports the concept of "static roles", which are
-a 1-to-1 mapping of Vault Roles to usernames in a database. The current password
-for the database user is stored and automatically rotated by Vault on a
-configurable period of time. This is in contrast to dynamic secrets, where a
-unique username and password pair are generated with each credential request.
-When credentials are requested for the Role, Vault returns the current
-password for the configured database user, allowing anyone with the proper
-Vault policies to have access to the user account in the database.
-
--> Please consult the specific database documentation on the left navigation or
-the table below under [Database Capabilities](#database-capabilities) to see if
-a given database backend supports static roles.
+With dynamic secrets, Vault generates a unique username and password pair for
+each unique credential request. Vault also supports **static roles** for
+some database secrets engines. Static roles are a 1-to-1 mapping of Vault roles
+to usernames in a database. With static roles, Vault stores, and automatically
+rotates, passwords for the associated database user based on a configurable
+period of time.
+
+When a client requests credentials for the static role, Vault
+returns the current password for the configured database user. With static
```

Comment:
I know that the wording in this didn't really change compared to what it was before, but we should not use "configured database user" to avoid the confusion against the user that's provided in `config/` i.e. the root credentials.

```suggestion
returns the current password for the database user that is mapped to this role. With static
```

Reply:
"that is mapped to this role" violates the style guide because "this" in potentially ambiguous. But I will rework the sentence to avoid "configured" :)